packages/app/src/app/graphql/types.ts

```diff
@@ -2071,7 +2071,10 @@ export type AllTeamsQuery = { __typename?: 'RootQueryType' } & {
       'personalWorkspaceId'
     > & {
         workspaces: Array<
-          { __typename?: 'Team' } & Pick<Team, 'id' | 'name' | 'avatarUrl'> & {
+          { __typename?: 'Team' } & Pick<
+            Team,
+            'id' | 'name' | 'avatarUrl' | 'joinedPilotAt'
```

Comment:
Added `joinedPilotAt` to the workspaces query because we use it in multiple places now. (This will get replaced with subscriptions after sanne's done)